jsk_2016_01_baxter_apc/euslisp/main-stow.l

```diff
@@ -39,7 +39,7 @@
           (ros::set-param (format nil "~a_hand/state" (send *ri* :arm-symbol2str arm)) (symbol2str state)))
         (:initialize
           (ros::ros-info "[main] ~a, ~a" arm state)
-          (setq tote-contents (send *ri* :get-tote-contents))
+          (setq tote-contents (get-tote-contents))
```

Comment:
I do not follow your idea because robot-interface should only contain methods which affect robot. Therefore `:set-target-bin` etc. should be separated from robot-interface.
i dont like make class  huge and hold many methods, all of not robot-related methods should be functions.
However, current discussion took me enormous time and power, so for this PR, i follow your idea.